utils/text/algorithms.py

```diff
@@ -0,0 +1,36 @@
+# Third Party (PyPI) Imports
+import numpy
+
+
+def levenshtein_distance(w1, w2):
+    """The Levenshtein distance algorithm that compares two words
+
```

Comment:
Let's also include the Wikipedia link:

`https://en.wikipedia.org/wiki/Levenshtein_distance`